firestore/integration_test_internal/src/util/integration_test_util.cc

```diff
@@ -37,7 +37,7 @@ struct TestFriend {
   static FirestoreInternal* CreateTestFirestoreInternal(App* app) {
 #if !defined(__ANDROID__)
     return new FirestoreInternal(
-        app, absl::make_unique<credentials::EmptyAuthCredentialsProvider>(),
+        app, /*database_id=*/"(default)",absl::make_unique<credentials::EmptyAuthCredentialsProvider>(),
```

Comment:
fill in `(default)` for test firestore for now, will adjust to accept databaseId later